templates/bot/ts/command-and-response/package.json

```diff
@@ -7,10 +7,10 @@
     "main": "./lib/index.js",
     "scripts": {
         "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
-        "dev": "nodemon -e ts --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
+        "dev": "nodemon --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
```

Comment:
I remember we exclude json file due to ignoring `.notification.localstore.json`, will this monitor the json file as well?

Reply:
How about using `nodemon.json` to control details? To ensure that only some required `json` files are in monitor scope.

Reply:
How about make `dev` script same to the `watch` script? Only the files under `src/` are watched.
Maybe user code contain some other kinds of files under `src` folder. (e.g. yml / html / ...... )